numba/_helperlib.c

```diff
@@ -825,143 +825,13 @@ static void traceback_add(const char *funcname, const char *filename, int lineno
     _PyErr_ChainExceptions(exc, val, tb);
 }
 
-/* Logic for raising an arbitrary object.  Adapted from CPython's ceval.c.
-   This *consumes* a reference count to its argument. */
-NUMBA_EXPORT_FUNC(int)
-numba_do_raise(PyObject *exc_packed)
-{
-    PyObject *exc = NULL, *type = NULL, *value = NULL, *loc = NULL;
+
+static
+void traceback_add_loc(PyObject *loc) {
```

Comment:
Would be good to include a comment about what this function does.

Reply:
Done in https://github.com/numba/numba/pull/6017/commits/07415ebafbe9ae43e20e824762ce6e3941b4f21c